docs/en/mutations.md

```diff
@@ -52,17 +52,17 @@ You can also dispatch mutations using objects:
 
 ``` js
 store.dispatch({
-  type: 'INCREMENT',
-  payload: 10
+  type: 'INCREMENT_BY',
+  payload: [ 10 ]
```

Comment:
If the goal is to encourage users to use an options style object so that payload 'args' are named in the dispatch,
that's achievable using something similar to the previous example. (there's a small typo on `mutations.md` L65)

The strange thing about `index.js` L83 is that it prevents you from using mutations with more than 2 arguments.
An implementation that would allow this would be more appropriate, but it's a breaking change:
```javascript
payload: [10, 'USD']
```
The named parameter example would be changed to: 
```javascript
payload: [{
  amount: 10,
  currency: 'USD'
}]
```

Our team ran into this when we wanted to switch to using Object-Style Dispatches in order to take advantage of the `silent` flag for certain actions.
We think the inability to use previously defined mutations with this style of dispatch is obstructive.